src/witan/workspace/handler.clj

```diff
@@ -21,15 +22,33 @@
             :tags [{:name "api", :description "some apis"}]}}}
 
    (GET "/by-owner" req
-        :summary "Just a test"
-        :components [db]
-        :query-params [owner :- s/Uuid]
-        (do
-          (log/debug req)
-          (w/by-owner owner (params-vector req :fields) db)))
+     :summary "Queries the workspaces by owner"
+     :components [db]
+     :query-params [owner :- s/Uuid]
+     (do
+       (log/debug req)
+       (w/by-owner owner (params-vector req :fields) db)))
```

Comment:
'Missing' owners will return an empty collection. One assumes we will have implemented the auth system to such a degree that only workspaces you have access to are returned. Not sure what you mean about DDoS re this particular route.